OpenRA.Mods.Common/Widgets/LabelWithHighlightWidget.cs

```diff
@@ -39,7 +39,7 @@ protected LabelWithHighlightWidget(LabelWithHighlightWidget other)
 		(string, bool)[] MakeComponents(string text)
 		{
 			var components = new List<(string, bool)>();
-			foreach (var l in text.Split(new[] { "\\n" }, StringSplitOptions.None))
+			foreach (var l in text.Split("\\n", StringSplitOptions.None))
```

Comment:
Can't this also be '\n' ?

Reply:
we need two characters

Reply:
scope creep but is seems the whole logic is failing. We fail to newline, and just remove the \n. It's not an issue by in default mods as we don't include \n in fluent

Reply:
Indeed, this is the 2-char string `\n`, it is not an actual newline character. I won't touch the existing logic as part of this PR, since it's quite unrelated to style changes.

However I think this logic might be intended, from inspection:
```
first <highlight>
second <highlight>
```
becomes "first _highlight_ second _highlight_"
but
```
first <highlight> second <not-highlight>
```
becomes
"first _highlight_ second \<not-highlight\>"

Since only one highlight per line is resolved.